doc/script_commands.txt

```diff
@@ -6231,6 +6231,15 @@ Example usage:
 
 ---------------------------------------
 
+*specialeffectnum(<effect number>, <num1>, <num2>{, <send_target>{, <unit id>{, <account id>}}})
+*specialeffectnum(<effect number>, <num1>, <num2>{, <send_target>{, "<NPC Name>"{, <account id>}}})
+
+Works same as specialeffect but also send effect numbers to client.
+For PACKETVER >= 20191127 support two numbers (num1, num2).
+For older packet versions only num1 supported.
```

Comment:
Any info on what num2 affects?

Reply:
num2 is high dword of num. Script engine not support int64, because this here num and num2. Inside packet its one int32 or one int64 depend on packet version